modules/authentication/src/handlers/local.ts

```diff
@@ -161,6 +161,19 @@ export class LocalHandlers implements IAuthenticationStrategy {
       this.verifyChangeEmail.bind(this),
     );
 
+    routingManager.route(
+      {
+        path: '/local/resend-verification',
```

Comment:
Make this `/local/new/send-verification` as it's not clear what sort of verification send this URI performs.
Update handler name to reflect this.

This should also only get registered whenever `this.sendVerificationEmail` is true.